src/EditLabelDialog/EditLabelDialog.jsx

```diff
@@ -24,7 +24,7 @@ export default function EditLabelDialog({
   onNew = () => {},
   onEdit = () => {},
   onClose = () => {},
-  labelDialogTitle = "Add Label",
```

Comment:
You need to update any uses of this:

e.g.https://github.com/IPG-Automotive-UK/react-ui/blob/336f7ab5cb9390c410f839a4d8031412f05df1bd/src/LabelSelector/LabelSelector.jsx#L279

![image](https://github.com/user-attachments/assets/1def9c1b-7265-4073-a4d5-d9b9aef80258)